src/System.Windows.Forms/src/System/Windows/Forms/TreeView.cs

```diff
@@ -863,6 +864,8 @@ public int ItemHeight
             }
         }
 
+        internal ToolTip KeyboardToolTip = new ToolTip { ReshowDelay = 500, InitialDelay = 500 };
```

Comment:
This looks like a field, and must reside at the top with the rest of fields, and match the naming guidelines

Reply:
1) It used in `TreeNodeCollection` - a separate class, not nested in TreeView, needs to use `internal` modifier
2) If so, when do we use a field with the `internal` modifier? I think, we should keep it as a property

Reply:
If this is a property then it should be like this:
```suggestion
        internal ToolTip KeyboardToolTip { get; } = new ToolTip { ReshowDelay = 500, InitialDelay = 500 };
```

Reply:
Also we should avoid magic numbers. Where did "500" come from?

Reply:
Sure, thanks
@SergeySmirnov-Akvelon, please rework it

Reply:
I would remove "500" magic numbers and just use a default `ToolTip`